lib/mockexpress.js

```diff
@@ -72,6 +72,10 @@ var MockExpress = function(route) {
 		},
 		'makeRequest': function(getMap) {
 			var requestObject = Object.create(null);
+
+			requestObject.cookies = [];
```

Comment:
`req.cookies` should be an object, according to the [API Reference](http://expressjs.com/en/api.html#req.cookies)